app/src/main/java/com/andrew/tasky/agenda/data/event/EventDao.kt

```diff
@@ -0,0 +1,18 @@
+package com.andrew.tasky.agenda.data.event
+
+import androidx.room.*
+import com.andrew.tasky.agenda.domain.models.AgendaItem
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface EventDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun upsert(event: AgendaItem.Event): Long
+
+    @Query("SELECT * FROM eventdto")
```

Comment:
A DTO is usually a network related object (that represents JSON payload). I'd call database models simply entities. So Instead of EventDto, this would be EventEntity here